src/system/ttytyp.319

```diff
@@ -184,6 +184,77 @@ MCONDX KA,{				;;; KA
 	TTDAAA 37,SPEED=9600		;T37
 	TTDAAA 40,SPEED=9600		;T40
 .ENDC ;}
+MCONDX KL,{				;;; KA
```

Comment:
Why add a new section to TTYTYP?  Line 187 is a new KL, but the comment says KA.  Line 348 is the old KL, now renamed to KX, but the comment says KL.

Reply:
I'll fix. I'll have to look at what I did. I think I did that because I wanted to use the KL build with terminals, and you hadn't defined any. But I can put it back because I've added the channels to the EX TTYTYP entry.  I suspect the "KA" comment for KL was there originally.  I remember fixing a few comments after machine definitions since they were wrong.

Reply:
So what I had done was this:  renamed your original KL entry to KX (to get it out of the way and preserve it); copied the KA entry to KL so that I could login to a KL system; added an EX entry once I had built/tested/deployed an EX system.  I forgot to get rid of the temporary KL entry and rename KX back to KL.  I've done that now.

Reply:
Thanks!